### DISCUSSION TITLE
Doesn't work

### DISCUSSION THREAD

**Discussion by pisticsumega** [jackweatherford/spotify-friend-activity#30] [General]
Added the extension both ways, and it did install itself both times, but nothing showing on the friends activity tab! I tried reloading with the small arrow icon, but to no avail.

**Comment by jackweatherford** [comment]
Your friends may all be using the Spotify website to listen to music, just like you are. If that is the case, then unfortunately you won't be able to see their activity. Basically, Spotify only records activity of Spotify desktop users, so that's all that is being displayed to this extension for now.
